drools-ansible-rulebook-integration-main/load_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Usage: ./load_test.sh
+# This script runs a series of test files. While you can confirm STDOUT, the results are redirected to result.txt through STDERR.
+# The result would be "testfile name, usedMemory (bytes), timeTaken (ms)"
+# -------- for example --------
+# 24kb_1k_events.json, 5246824, 192
+# 24kb_10k_events.json, 5221216, 648
+# 24kb_100k_events.json, 5285568, 3958
+# 24kb_1m_events.json, 5282712, 33741
+# 24kb_1k_events_unmatch.json, 5158648, 122
+# 24kb_10k_events_unmatch.json, 5166952, 364
+# 24kb_100k_events_unmatch.json, 5249480, 1682
+# 24kb_1m_events_unmatch.json, 5291136, 12940
+
+# Truncate (or create) result.txt so we start fresh each time
+> result.txt
+
+# List of JSON files to process
+files=(
```

Comment:
Great work, I just wonder if it makes sense to run all these files in this script by default. Maybe you can add an argument with possible values 1k/10k/100k/1m (with default 1k) and run the 3 scripts with the specified size?